innovation.game.php

```diff
@@ -10486,6 +10505,58 @@ function stInterInteractionStep() {
                     }
                     break;
                 
+				// id 123, Artifacts age 1: Ark of the covenant
+                case "123N1A":
+					$players = self::loadPlayersBasicInfos();
+					
+					if ($n > 0)
+					{ // Unsaid rule: the player must have at least one card to show from his hand, else, the effect can't continue
+                        $color = self::getGameStateValue('color_last_selected');
```

Comment:
Can you rename this variable to "return_color" for additional clarity? This will also free up the color variable in case we need to use it in a for loop.

Reply:
Yes.